lms/djangoapps/courseware/tests/test_password_history.py

```diff
@@ -199,7 +199,7 @@ def test_staff_password_reset_reuse(self):
         staff_email, _ = self._setup_user(is_staff=True)
         user = User.objects.get(email=staff_email)
 
-        err_msg = 'You are re-using a password that you have used recently. You must have 2 distinct passwords'
+        err_msg = 'You are re\\\\u002Dusing a password that you have used recently. You must have 2 distinct passwords'
```

Comment:
does this required even when you are using `js_escaped_string` before passing to react code.

Reply:
@tasawernawaz Yes, it is always required if we escape the string using 'js_escaped_string', passing to reactJS doesn't matter, it just needs a escaped string otherwise mako will rais error.